components/signersForm.tsx

```diff
@@ -552,9 +567,14 @@ const SignersForm: FC<{
                 </div>
               </div>
               {/* @ts-ignore*/}
-              {!!errors.proposalDuration &&
-                // @ts-ignore
-                renderError(errors.proposalDuration)}
+              {!!errors.proposalDuration
+                ? // @ts-ignore
+                  renderError(errors.proposalDuration)
+                : currentDuration < PROPOSAL_DURATION_WARNING
+                ? renderWarning(
+                    "Proposal duration is low, you may not be able to resolve proposals"
```

Comment:
when proposal is expired, users still can resolve the proposals. Therefore, I suggest s/resolve/execute.